src/keymap.rs

```diff
@@ -23,6 +24,34 @@ impl Keymap {
     self.term.insert(key, event);
   }
 
+  pub fn resolve_key(&self, scope: Scope, app_event: &AppEvent) -> Vec<&Key> {
+    let map = match scope {
+      Scope::Procs => &self.procs,
+      Scope::Term | Scope::TermZoom => &self.term,
+    };
+    let mut vec = Vec::new();
+    for (k, v) in map.iter() {
+      if v == app_event {
+        vec.push(k)
```

Comment:
My initial implementation was like that, but unfortunately that won't work because a single `AppEvent` can be bind by multiple keys. (Eg: `NextProc` is bind using both Down button as well as the key binding `j`).

So we need to return all the default keybindings in a vector so that it can be compared later - We want to just find out the keybinding which we have specifically overriden.

Reply:
Let's just show the first binding we find. The help window just shows how to perform some basic actions. Later I plan to add a bigger window which will show all the bindings.

Reply:
> The help window just shows how to perform some basic actions.

Ah, okay.

> Let's just show the first binding we find. 

I think in that case, probably we should just leave the current code as it is. If we change it to use this patch, then the interface is going to be show different key bindings on different mproc invocation for actions which has multiple key bindings (because it's backed by HashMap instead of IndexMap and I don't think it's worth fixing that) and I think that would be confusing. WDYT ?

Given the situation, what do you think about closing the PR ?

Reply:
Yeah, showing a random key is not ideal. But it would still be the case for multiple custom bindings, if we exclude default bindings. Ideally we should choose either first or last binding, but that would require different logic. Feel free to close the PR if you are not planning to deal with this logic and I'll take care of it later at some point. But thanks for the effort anyway!

Reply:
> But it would still be the case for multiple custom bindings, if we exclude default bindings.

Yeah, right. Multiple bindings at different location would be still buggy with the current patch.

> Ideally we should choose either first or last binding, but that would require different logic.

Yeah, it should be easy to fix. But I think I would just prefer the current behavior instead. Thanks!